.gitignore

```
@@ -76,4 +76,4 @@ target/
 .DS_Store
 
 # written by setuptools_scm
-*/_version.py
+*/*/_version.py
```

Ah yes, this was missed when I moved to src layout. Actually it should probably be:

```suggestion
src/*/_version.py
```